### ISSUE TITLE
Banner on all pages are not scaling to page width

### ISSUE THREAD

**Issue by neerajskydiver** [drupal-bangalore/drupalcamp-bangalore#1]
![image](https://cloud.githubusercontent.com/assets/7127061/8511801/074420d2-2344-11e5-9d98-c3515ffe7b16.png)

**Comment by hussainweb** [comment]
@neerajskydiver, If banner did scale to page width, it would look stretched. I would vote to keep it this way until we find a good alternative.

**Comment by neerajskydiver** [comment]
This can be fixed using CSS, if banner image is as background property. Let me know if you need help.

**Comment by ramrudra** [comment]
@neerajskydiver Are you talking something like this? With image positioning?
![screen shot 2015-07-05 at 18 53 19](https://cloud.githubusercontent.com/assets/3307301/8511896/3f8417ec-2347-11e5-96a3-f33f7f0975ac.png)

**Comment by neerajskydiver** [comment]
no

**Comment by neerajskydiver** [comment]
Try this 

.l-header .l-branding{background:url(/sites/all/themes/drupalcamp/images/banner.jpg) transparent center bottom no-repeat;background-size:auto 100%;min-height:550px;}

to

.l-header .l-branding{background:url(/sites/all/themes/drupalcamp/images/banner.jpg) transparent no-repeat;height:550px; background-position: center center; background-size: cover;    overflow: hidden;}

**Comment by ramrudra** [comment]
Yes. Exactly the same as I mentioned above. The image needed to be positioned in the above attached image.

**Comment by ramrudra** [comment]
It finally comes like this 
![screen shot 2015-07-05 at 18 56 07](https://cloud.githubusercontent.com/assets/3307301/8511904/b2ad15a2-2347-11e5-8cb1-ebe49d6aff1f.png)

**Comment by neerajskydiver** [comment]
This looks good. Check with Hussain if we can remove Diamond Sponsors logo and remove the min-height property. Instead we can give fixed height of around 350px.

**Comment by hussainweb** [comment]
@neerajskydiver, @ramrudra, sorry it just looks too stretched to me. I'd prefer it the current way instead.

I do think we can reduce sponsor logo size.

**Comment by neerajskydiver** [comment]
![image](https://cloud.githubusercontent.com/assets/7127061/8511936/1516675a-234a-11e5-8a97-ef6dce66849b.png)
This is how it looks in chrome. Image is not stretched. Its just fitting to page width. compare the banner fit with actual image - http://drupalcampbangalore.org/sites/all/themes/drupalcamp/images/banner.jpg

**Comment by ramrudra** [comment]
@neerajskydiver I'm assuming we will have Diamond sponsor logos in the centre which might look odd.

**Comment by neerajskydiver** [comment]
@ramrudra you should check this with @hussainweb .

**Comment by neerajskydiver** [comment]
Banner not scaling to full page width yet. Are we keeping as it is?

**Comment by hussainweb** [comment]
I am in favour of keeping it as it is. I don't like images when they get stretched. It seems unprofessional and even if people don't realize it is stretched, there is a feeling that there is something awkward about the image.

**Comment by JayaJayanth** [comment]
Sorry hussain, yes issue #8 was a duplicate. keeping background-size as 100% doesnt seem to stretch the image as i see it in firefox. Has that been tried?
